GamingSupervisor/GamingSupervisor/ReplayAnalyzer.cs

```diff
@@ -326,6 +336,8 @@ private void HandleHighlight()
 
         private void HandleGamePlay()
         {
+            #region Hero introduction
```

Comment:
I don't think we should be using regions. If code should be divided up, it should be put into functions.

Reply:
Yeah, you are right. I will make a function.